Auth0/WebAuth.swift

```diff
@@ -118,6 +118,15 @@ public protocol WebAuth: Trackable, Loggable {
     func scope(_ scope: String) -> Self
 
     /**
+      Provider scopes for oauth2/social connections. e.g. Facebook, Google etc
+
+     - parameter connectionScope: oauth2/social scope list: `user_friends email`
```

Comment:
@hzalaz Can I suggest there should be a method to reset parameters, if you are using a common instance of `WebAuth`, parameters are cumulative with no way to remove them. 
This is most relevant with the requirement of connection specific params like `connectionScopes` and `login_hint`.

You can of course create a new instance each time but it is less expensive to `[:]` than it is init a new object. 

Thoughts?